shippable_adapter.py

```diff
@@ -79,20 +79,21 @@ def __put(self, url, body):
         }
         while True:
             try:
+                err = None
```

Comment:
This was missing and hence the variable `err` in line 97 (of this file) was undefined.

![selection_300](https://cloud.githubusercontent.com/assets/6316230/14784978/eb95015c-0b15-11e6-88a5-ca8a6c87dccd.jpg)